tools/cli/flags.go

```diff
@@ -299,6 +299,7 @@ const (
 	FlagIsolationGroupSetDrains           = "set-drains"
 	FlagIsolationGroupJSONConfigurations  = "json"
 	FlagIsolationGroupsRemoveAllDrains    = "remove-all-drains"
+	FlagSearchAttribute                   = "search-attribute"
```

Comment:
nit: maybe use `search-attr` to be consistent since we have `get-search-attr` flag